src/main/kotlin/com/cosmotech/api/utils/SecurityUtils.kt

```diff
@@ -7,7 +7,10 @@ import org.springframework.security.core.Authentication
 import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.security.oauth2.server.resource.authentication.BearerTokenAuthentication
 
-fun getCurrentAuthentication(): Authentication? = SecurityContextHolder.getContext().authentication
+fun getCurrentAuthentication(): Authentication? {
```

Comment:
Could you please remove this and test again?
Apparently, it has no effect for the moment, so there's no need to keep it ;)